content/02-getting-started/01-installing-the-cardano-node.mdx

```diff
@@ -17,6 +17,88 @@ the source code using either of the following:
 Once you have installed the node, you need to
 [specify the configuration parameters](https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/understanding-config-files.md/).
 
+### How to Run the Cardano node and CLI using Docker
+There is also a Docker image available that you can use. 
+
+#### Setting up Docker
+
+1. Download and install Docker/Docker Desktop from the [Docker site](https://docs.docker.com/get-docker/).
+2. Ensure you have allocated enough RAM to Docker (we recommend setting the RAM to 8GB in the Docker Desktop requirements) and start the Docker daemon, as described in the Docker instructions.
+
+#### Setting up the Cardano node
+1. Download the correct `cardano-node` image:
+
+``docker image pull inputoutput/cardano-node:<TAG>``
+
+where `<TAG>` is the tag for the version that you require (e.g. `latest` for the most recent stable version, or `1.27.0` for node version 1.27.0).
+
+2. Create local `data` and `node-ipc` volumes:
+
+```
+docker image create data
+docker image create node-ipc
+```
+
+## Running the Cardano node
+
+1. Run a passive node that is connected to the correct network. For example, for a `mainnet` node:
+
+``docker run -e NETWORK=mainnet -v node-ipc:/ipc -v data:/data inputoutput/cardano-node``
+
+This creates a persistent docker environment for the node, where `/ipc` in the Docker container is connected to the logical `node-ipc`volume, and `/data` is connected to the `data` volume.  Note that you should change `NETWORK=mainnet` if you are connecting to a different network (eg a testnet).  
+
+You may also run the node with specific parameters:
+
+``docker run -v node-ipc:/ipc -v data:/data inputoutput/cardano-node run --help``
+
+### Passing Explicit Configuration Parameters
+If there is no pre-defined network configuration, you will need to download the specified configuration files, copy these to the persistent Docker volume, and pass the parameters on the command line:
+
+``docker run -e NETWORK=mainnet -v node-ipc:/ipc -v data:/data inputoutput/cardano-node run --config ... --topology ... --``
+
+## Running Cardano CLI commands
+You can now run normal Cardano CLI commands, for example,
+```
+export CLI='docker run -it --entrypoint cardano-cli -e NETWORK=mainnet -e CARDANO_NODE_SOCKET_PATH=/ipc/node.socket -v node-ipc:/ipc inputoutput/cardano-node'
+$CLI version
+$CLI query tip --mainnet
+$CLI transaction build-raw ... --mainnet
+```
+You need to specify `CARDANO_NODE_SOCKET_PATH` to point to the correct location in the container (`/ipc/node.socket` is the standard location).
+
+## Running a Shell in the Docker Container
+To run a shell in the container (so that you can inspect or change settings, for example), use the `bash` or `sh` entry point.
+
+``docker run -it --entrypoint bash -v node-ipc:/ipc -v data:/data inputoutput/cardano-node``
```

Comment:
An alternative method that means you can still see node logs and when shell is exited it won't kill the node:

Start node container with a name:
`docker run --name cardano-node -e NETWORK=mainnet -v cardano-node-ipc:/ipc -v cardano-node-data:/data inputoutput/cardano-node`

In a separate terminal instance, run shell in existing container:
`docker exec -it cardano-node bash`